types/testingutils/beacon_node_versioned.go

```diff
@@ -5,46 +5,50 @@ import (
 
 	"github.com/attestantio/go-eth2-client/api"
 	apiv1capella "github.com/attestantio/go-eth2-client/api/v1/capella"
+	apiv1deneb "github.com/attestantio/go-eth2-client/api/v1/deneb"
 	"github.com/attestantio/go-eth2-client/spec"
-	"github.com/attestantio/go-eth2-client/spec/bellatrix"
 	"github.com/attestantio/go-eth2-client/spec/capella"
+	"github.com/attestantio/go-eth2-client/spec/deneb"
 	"github.com/attestantio/go-eth2-client/spec/phase0"
 	ssz "github.com/ferranbt/fastssz"
 
 	"github.com/bloxapp/ssv-spec/types"
 )
 
 const (
+
+	//TO-DO: Define Deneb Fork Epoch
+	ForkEpochPraterDeneb = 225431
```

Comment:
I'm not sure what's the significance of that, but that fork epoch is wrong, I think this is the right one:
https://github.com/ethereum/execution-specs/blob/559ba54829a5d726ca95fe5701486f7bd303bf7f/network-upgrades/mainnet-upgrades/cancun.md

Reply:
@MatheusFranco99 
This may be a very bad bug in our value check... not sure
are you debigging this?
do you want me to look at this with you?

Reply:
I'm looking into it.
But, basically it's triggered by [the first duty value check](https://github.com/bloxapp/ssv-spec/blob/main/ssv/value_check.go#L20-L21) which uses [some network functions](https://github.com/bloxapp/ssv-spec/blob/main/types/beacon_types.go#L116-L199).

Reply:
So this can only be changed around the time the fork happens
The implementation shouldn't use this value